templates/general.js

```diff
@@ -85,17 +85,29 @@ ${name};
     return `
 {
   "name": "${moduleName}",
+  "title": "${moduleName.split('-').map(word => word[0].toUpperCase() + word.substr(1)).join(' ')}",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
   "scripts": {
     "test": "echo \\"Error: no test specified\\" && exit 1"
   },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/${authorGithub}/${moduleName}.git",
+    "baseUrl": "https://github.com/${authorGithub}/${moduleName}"
+  },
   "keywords": [
     "react-native"
   ],
-  "author": "",
-  "license": "",
+  "author": {
+    "username": "${authorGithub}",
+    "name": "${authorName}",
+    "email": "${authorEmail}"
+  },
+  "license": "Apache-2.0",
```

Comment:
Nitpick: Do we need to hardcode the license? Could we get that as a command-line parameter or default value as well?

Reply:
Sure. Fair point I'll make it configurble as well. 💯

Reply:
Done.